ballerina-to-openapi/build.gradle

```
@@ -132,3 +132,27 @@ compileJava {
         classpath = files()
     }
 }
+
+publishing {
+    publications {
+        mavenJava(MavenPublication) {
+            groupId project.group
```

Can we change the group here? The project group is `io.ballerina`. I think it is better to publish it under `io.ballerina.openapi`
```suggestion
            groupId "io.ballerina.openapi"
```

---

`Persist tool` is publish to `io.ballerina`.
https://github.com/ballerina-platform/persist-tools/blob/a487cbc0c085e572d042a74e851d1a2e51e4da3a/persist-cli/build.gradle#L151

Don't we need to have a same group id among the tools?

---

That is the entire tool right? This is just a part of the tool

---

ok, will update that